### PR TITLE
docs: add proposals related to Konveyor Move2Kube

### PR DESCRIPTION
**Pull request by HarikrishnanBalagopal**
draft

### CODE REVIEW

**Comment by HarikrishnanBalagopal** [comment]
Draft

**Comment by aliok** [comment]
@HarikrishnanBalagopal Thanks for your proposal.

Can you ping us when you are ready with your proposal draft?

Also, we would need a LGTM from one of the maintainers listed here: https://github.com/konveyor/community/blob/main/MAINTAINERS.md#maintainers

**Review by caniszczyk — APPROVED** [review]

**Comment by kmehant** [comment]
@aliok The PR is ready. 

Requesting for PR approval from one of @jwmatthews @JonathanR19 @rromannissen @savitharaghunathan @tamintheclouds @tsanders-rh, This proposal covers problems statements for [Konveyor Move2Kube](https://github.com/konveyor/move2kube). Thank you.

**Pull request merged**
